datalad/core/distributed/tests/test_clone.py

```diff
@@ -816,6 +816,15 @@ def _test_ria_postclonecfg(url, dsid, clone_path):
                                                     if url.startswith('http')
                                                     else "store-storage"))
 
+    # finally get the plain git subdataset.
+    # Clone should figure to also clone it from a ria+ URL
+    # (subdataset-source-candidate), notice that there wasn't an autoenabled ORA
+    # remote, but shouldn't stumble upon it, since it's a plain git.
+    res = riaclone.get(op.join('subdir', 'subgit', 'testgit.txt'))
+    assert_result_count(res, 1, status='ok', type='dataset', action='install')
+    assert_result_count(res, 1, status='notneeded', type='file')
```

Comment:
This line is failing with

```
AssertionError: Got 0 instead of 1 expected results matching
  {
   "status": "notneeded",
   "type": "file"
  }
Inspected 2 record(s):
  [
   {
    "action": "install",
    "contains": [
     "/tmp/datalad_temp__test_ria_postclonecfgg92yk3jn/subdir/subgit/testgit.txt"
    ],
    "message": [
     "Installed subdataset in order to get %s",
     "/tmp/datalad_temp__test_ria_postclonecfgg92yk3jn/subdir/subgit/testgit.txt"
    ],
    "path": "/tmp/datalad_temp__test_ria_postclonecfgg92yk3jn/subdir/subgit",
    "source": {
     "default_destpath": "67170c89-8f99-11ea-a17b-42010a1400f3",
     "giturl": "file:///tmp/datalad_temp_oz2uaxsu/671/70c89-8f99-11ea-a17b-42010a1400f3",
     "source": "ria+file:///tmp/datalad_temp_oz2uaxsu#67170c89-8f99-11ea-a17b-42010a1400f3",
     "type": "ria",
     "version": null
    },
    "status": "ok",
    "type": "dataset"
   },
   {
    "action": "get",
    "message": "no dataset annex, content already present",
    "path": "/tmp/datalad_temp__test_ria_postclonecfgg92yk3jn/subdir/subgit/testgit.txt",
    "refds": "/tmp/datalad_temp__test_ria_postclonecfgg92yk3jn",
    "status": "notneeded"
   }
  ]
```

<https://travis-ci.org/github/datalad/datalad/jobs/683807836#L1807>

The assertion looks fine.  I think that "get" result should have a file action:

```diff
diff --git a/datalad/distribution/get.py b/datalad/distribution/get.py
index 828f10a17..8c7a66abc 100644
--- a/datalad/distribution/get.py
+++ b/datalad/distribution/get.py
@@ -531,7 +531,9 @@ def _get_targetpaths(ds, content, refds_path, source, jobs):
         for r in results_from_paths(
                 content, status='notneeded',
                 message="no dataset annex, content already present",
-                action='get', logger=lgr,
+                action='get',
+                type='file',
+                logger=lgr,
                 refds=refds_path):
             yield r
         return
```

Reply:
Testing locally.  Will push after.

Reply:
Hmm, I can't push to this branch:

```
To github.com:mih/datalad.git
 ! [remote rejected]     bf-4485 -> bf-4485 (permission denied)
error: failed to push some refs to 'git@github.com:mih/datalad.git'
```

I've pushed to a branch with the same name to my remote, if someone wants to bring it in.

Reply:
I don't know what changed. Nobody can push to my fork anymore :(

Reply:
Merged and pushed, thx.

Reply:
> I don't know what changed. Nobody can push to my fork anymore :(

A while ago @bpoldrack mentioned that he couldn't push to one of my branches.  Since then, it seems to have worked fine (at least for other people), so perhaps it's a weird github glitch that will resolve itself :/

Reply:
Interestingly, this failure didn't happen for me locally before I pushed. But I agree with the fix. Right thing to do regardless of whether it leads to failure.

Reply:
> Interestingly, this failure didn't happen for me locally

Oh, odd.  As you say, it doesn't really matter, but FTR I was able to trigger it locally.